resources/js/Pages/SinkStatus.vue

```diff
@@ -575,6 +589,19 @@ onMounted(() => {
       </template>
     </v-data-table-server>
 
+    <v-dialog v-model="docsDialog" max-width="90%" max-height="80%">
+        <v-card>
+        <v-toolbar color="primary">
```

Comment:
Vi bruker to innrykk på html-delen av koden. Vetkje hvorfor det gikk gjennom eslint :-/

Reply:
Det var noe visual code klarte å komme seg rundt på et eller annet vis....

Reply:
fixed